deploy/azure/infra/main.tf

```diff
@@ -17,6 +17,34 @@ resource "azurerm_resource_group" "default" {
   tags     = module.default_label.tags
 }
 
+module "networking" {
+  source                  = "git::https://github.com/amido/stacks-terraform//azurerm/modules/azurerm-hub-spoke"
+  enable_private_networks = true ## NOTE setting this value to false will cause no resources to be created !!
+  network_details         = var.network_details
+  resource_group_name     = azurerm_resource_group.default.name
+  resource_group_location = azurerm_resource_group.default.location
+  create_hub_fw           = var.create_hub_fw
+  create_fw_public_ip     = var.create_fw_public_ip 
+  create_private_dns_zone = var.create_private_dns_zone
+  dns_zone_name           = module.default_label.id
+}
+
+module "vmss" {
```

Comment:
Maybe we need to add something to control this if we don't have private networks ?

Reply:
Pushed an update to this, it now uses the  `create_private_dns_zone` variable to determine what is sets the value as.
`dns_zone_name           = var.create_private_dns_zone ? module.default_label.id : null`